Exercices/JS04/index.js

```diff
@@ -0,0 +1,5 @@
+"use strict"; 
+alert("Enter two numbers to see if the first is greater than the second");
+let fristNumber = prompt("Insert frist number:");
```

Comment:
As a good practice, make sure to convert text to numbers going forward 🙂 

Apart from that good job @lf-alonso